doc/source/whatsnew/v1.0.0.rst

```diff
@@ -175,6 +175,7 @@ Groupby/resample/rolling
 -
 -
 - Bug in :meth:`DataFrame.groupby` not offering selection by column name when ``axis=1`` (:issue:`27614`)
+- Bug in :meth:`DataFrameGroupby.agg` not able to use lambda function for ``aggfunc`` argument (:issue:`27519`)
```

Comment:
I think this problem is just for named aggregation, right? (The `**kwargs`-style?)

Reply:
this problem indeed occurs in `pd.NamedAgg`, but also in this kind:
```python
df.groupby(by="kind").agg(
      height_sqr_min=("height", lambda x: np.min(x ** 2)),
      height_max=("height", "max"),
      weight_max=("weight", "max"),
)
```
i am actually not sure how to properly describe this issue, suggestions will be very welcome! @TomAugspurger

Reply:
That kind is "named aggregation". Anything with `.agg(name=(selection, function), ...)` is named aggregation.

So instead of "for `aggfunc` argument", say "with named aggregation".

Reply:
clear! thanks for your quick reply! appreciate a lot!